run-tests.sh

```diff
@@ -10,7 +10,7 @@
 export PYTEST_ADDOPTS='docs tests invenio_app'
 
 pydocstyle invenio_app tests docs && \
-isort -rc -c -df && \
+isort **/*.py --check-only --diff && \
```

Comment:
In case I am not the only one: depending on a developer configuration they may have their virtualenv in the code folder (via `.venv/` for instance). We don't want to run the checks on that folder though since it contains code that is outside our control. Only targeting `invenio_app`, `tests` and `docs` would be great.

Reply:
I agree, we should foucs on ``invenio_app``, and ``tests``. I don't think we need the docs though.